### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by flying-sheep**
Also has a placeholder for version info so people know we care about anndata version info

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  validations:
+    required: true
+- type: textarea
+  id: traceback
+  attributes:
+    label: Error output
+    description: Produced by the code above. Leave empty if the issue is unexpected behavior instead of an error.
+    render: pytb
+  validations:
+    required: false
```
Comment (ivirshup) [inline .github/ISSUE_TEMPLATE/bug-report.yml:45]:
```suggestion
- type: textarea
  id: Report
  attributes:
    label: Report
    description: Describe the bug you encountered, and what you were trying to do. Please use [github markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) features for readability.
  validations:
    required: true
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- type: textarea
+  id: versions
+  attributes:
+    label: Versions
+    description: Which version of anndata and other related software you used.
```
Comment (ivirshup) [inline .github/ISSUE_TEMPLATE/bug-report.yml:50]: Can we specifically request the output of `session_info` here?

**Review by ivirshup — COMMENTED** [review]
Thanks!

Made a suggestion and requested a change.

I would like to keep the template for bugs more simple, just because I don't think the more structured approach works for many of the bugs I report.

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (flying-sheep) [reply .github/ISSUE_TEMPLATE/bug-report.yml:45]: In my experience, that leads to too unspecific reports. I’d rather have people write “nothing” in the code field in the rare case that it’s not about code but installation or so than have people forget to paste a stacktrace.

**Comment by flying-sheep** [comment]
> My idea would look like this: [flying-sheep/playground/issues/new?assignees=&labels=Bug+%F0%9F%90%9B&projects=&template=anndata-bug-report.yml](https://github.com/flying-sheep/playground/issues/new?assignees=&labels=Bug+%F0%9F%90%9B&projects=&template=anndata-bug-report.yml)
> 
> I’d prefer that to your version. Can you live with that? 😉

**Comment by flying-sheep** [comment]
@ivirshup said he’s fine with this

**Pull request merged**